### DISCUSSION TITLE
Run BLE JoyStick in background after pairing.

### DISCUSSION THREAD

**Discussion by Agilulfo** [ExpressLRS/ExpressLRS#3012] [Ideas]
Hey I've just discovered and tested the BLE joystick feature. 
It works very well, however I was initially confused when I've realized I could not exit the screen that says "joystick running".

It would be nice if the script would run only to pair the elrs transmitter and the bluetooth connection would keep working after it once the script is closed.

I would love to turn on my radio, pick the "pc-simulator" model and have the radio to automatically connect with my pc via bluetooth without having to run the script every time.

If possible would be nice to be able to have only the BLE connectio on to reduce battery consumption (It's not clear if this is already the case or not)

**Comment by pkendall64** [comment]
Press and hold the return button to exit the script and the BLE continues to run

**Reply by Agilulfo** [reply]
oh great, this works well! I was simply clicking return and that did not work. holding does!

How about having the radio automatically connect via bluetooth without having to run the script?  do you know if that is possible?

**Reply by pkendall64** [reply]
No, not possible with the code the way it currently is, not without making code changes.
